guide/14-deep-learning/How_MaskRCNN_works.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Please remove this sentence 'An iterative subdivision algorithm at selected locations is used to make point-based segmentation predictions' instead we should write 'Image segmentation methods often predict labels on a low-resolution regular grid, e.g., 1/8-th of the input. To make it actual size of an input image, interpolation takes place while in pointrend iterative subdivision algorithm is used to make it actual size of an image by predicting labels of the point by a trained small neural network'.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/881/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Thanks! Updated.</p>